docs/conversion.md

```diff
@@ -4,7 +4,7 @@ This document outlines all possible conversion details regarding `docker-compose
 
 The current table covers all **current** possible Docker Compose keys.
 
-__Note:__ due to the fast-pace nature of Docker Compose version revisions, minor versions such as 2.1, 2.2, 3.3 are not supported until they are cut into a major version release such as 2 or 3.
+__Note:__ due to the fast-pace nature of Docker Compose version revisions, minor versions such as 2.1, 2.2 are not supported until they are cut into a major version release such as 2 or 3.
```

Comment:
Go ahead and completely remove this line.. We should simply state that we don't support anything 3.4 and above at the moment.